lib/reporting/reports/payments/itemised_payment_totals.rb

```diff
@@ -6,7 +6,9 @@ module Payments
       class ItemisedPaymentTotals < Base
         def columns
           {
-            payment_state: proc { |orders| orders.first.payment_state },
+            payment_state: proc { |orders|
+              I18n.t("js.admin.orders.payment_states.#{orders.first.payment_state}")
```

Comment:
I think I'd prefer to use the keys under `spree.payment_states`.

This `js.` key is more for front-end use.